ngraph/python/tests/test_onnx/test_ops_reduction.py

```diff
@@ -17,7 +17,34 @@
 import onnx
 import pytest
 
-from tests.test_onnx.utils import run_node
+from tests.test_onnx.utils import run_node, unstrict_xfail_issue_35925, strict_xfail_issue_35925
+
+reduce_max = "ReduceMax"
+reduce_min = "ReduceMin"
+reduce_mean = "ReduceMean"
+reduce_sum = "ReduceSum"
+reduce_prod = "ReduceProd"
+
+reduce_data = np.array([[[5, 1], [20, 2]], [[30, 1], [40, 2]], [[55, 1], [60, 2]]], dtype=np.float32)
+reduce_axis_parameters = [
+    # (operation, keepdims - True/False, axis)
```

Comment:
I'm not sure how this comment relates to the following code...

Reply:
Now this comment is deprecated - it was related to previous version  - fixed